content/applications/hr/payroll.rst

```diff
@@ -413,36 +564,53 @@ field.
 
 .. image:: payroll/input-type-new.png
    :align: center
-   :alt: Create a new Input Type.
+   :alt: A new Input Type form filled in..
+
+Salary attachment types
+-----------------------
+
+Salary attachments, also thought of as wage garnishments, is money taken out of a payslip for
```

Comment:
```suggestion
Salary attachments, also thought of as wage garnishments, are money taken out of a payslip for
```
If "are money" doesn't work on its own, I would try something like "are portions or earnings"

Reply:
I like "are portions of earnings" - so I combined what you suggested =)